addons/html_editor/static/src/editor/core/delete_plugin.js

```diff
@@ -0,0 +1,726 @@
+/** @odoo-module */
+
```

Comment:
still complicated, but this file is really much more maintainable than before, in my opinion